src/auto-evo/simulation/SimulationCache.cs

```diff
@@ -39,7 +39,7 @@ public EnergyBalanceInfo GetEnergyBalanceForSpecies(MicrobeSpecies species, Biom
 
             // Auto-evo uses the average values of compound during the course of a simulated day
             cached = ProcessSystem.ComputeEnergyBalance(species.Organelles, biomeConditions, species.MembraneType,
-                species.PlayerSpecies, worldSettings, CompoundAmountType.Average);
+                species.PlayerSpecies, !species.IsBacteria, worldSettings, CompoundAmountType.Average);
```

Comment:
The `!` on this line is now wrong because the parameter is `isBacteria` so negating this value when passing it in is now wrong.